src/workerd/io/compatibility-date.capnp

```diff
@@ -254,4 +254,16 @@ struct CompatibilityFlags @0x8f8c1b68151b6cef {
   # WHATWG URL Standard, leading to a number of issues reported by users. Unfortunately,
   # the specCompliantUrl flag did not contemplate the redirect usage. This flag is
   # specifically about the usage in a redirect().
+
+  workerdExperimental @24 :Bool
```

Comment:
Went with `workerdExperimental` just in case we, at some point, wanted to have a similar separate catch-all flag for internal-only experimental. But I have no strong preference on the naming.

Reply:
Hmm, I don't see a reason to have a separate flag for internal code, I think we'd use the same one.

Reply:
Well, just `experimental` doesn't work because it name clashes with the annotation. I think I'll just keep it as `workerdExperimental` and we can change it later if we absolutely need to.